README.md

```diff
@@ -107,6 +107,8 @@ npm run lint:fix
 ### Storybook
 To start storybook locally run `npm run storybook`.
 
+To view the production storybook visit [chromatic](https://main--60e472b3f21a0d003bd71f2f.chromatic.com). To view the latest published storybook on a given branch, visit `https://<branch>--60e472b3f21a0d003bd71f2f.chromatic.com`. Replace <branch> with the designated branch.
```

Comment:
Link is 404 for me.

Reply:
You shouldn't get a 404 once it's merged.